charts/ratify/values.yaml

```diff
@@ -23,7 +23,7 @@ ratifyTestCert: |
   Ibo/J0urezLK/M8/QeMkkhY1WClsAi+xLhVWUw==
   -----END CERTIFICATE-----
 cosign:
-  enabled: false
+  enabled: true
```

Comment:
do probably want to leave cosign disabled by default.

Reply:
Wouldn’t it be better to make this change later. I’m leaning towards having co-sign enabled by default maybe until RC and decide with the community. Also catches any issues that this might bring up early on.

Reply:
Sure, i ll do more testing to make sure following scenarios work with cosign configured.
1.  Image with only notary signature
2. Image with only cosign signature
3. Image with both notary and cosign signatures

Reply:
to set default to true, we need provide a default public key, Is there a keyvault to use for storing private/public key?

Reply:
Yes, we have a keyvault we are using for the net-monitor repo and can use that for cosign as well.